htdocs/adherents/class/api_members.class.php

```diff
@@ -178,7 +178,7 @@ public function post($request_data = null)
         if ($member->create(DolibarrApiAccess::$user) < 0) {
             throw new RestException(500, 'Error creating member', array_merge(array($member->error), $member->errors));
         }
-        return $member->id;
+        return $this->get($member->id);
```

Comment:
Making here a sql request to load to return to user a record we have create with data provided by user is not expected behaviour.
For create, current practice in dolibarr API is to return the ID of creation to avoid SQL overload.

Reply:
Ok, I understand. Happy to follow Dolibarr's practices but I still have empty bodies. Am I the only one having the issue? If yes, I will investigate PHP / server versions.

Empty bodies and [the related open issue](https://github.com/Dolibarr/dolibarr/issues/12761) were my motivation for the PR.

Just a question, if this is a common issue, to avoid an extra SQL request, could we use $this->_cleanObjectDatas() instead of $this->get()?